trin-core/src/utp/trin_helpers.rs

```diff
@@ -51,14 +51,17 @@ pub struct UtpAccept {
     pub message: Vec<(Vec<u8>, Vec<u8>)>,
 }
 
-// This is not in a spec, this is just for internally tracking for what portal message
-// negotiated the uTP stream
-#[derive(Debug, Clone)]
-pub enum UtpMessageId {
+/// Used to track which stream to which overlay request correspond
+#[derive(Debug, Clone, PartialEq)]
+pub enum UtpStreamId {
```

Comment:
The name `UtpStreamId` is a bit confusing to me, I think because `ConnId` is already acting as an identifier for streams.
I wonder if `UtpStreamType` or `UtpStreamOrigin` would be a more descriptive name.

Reply:
`ConnId` is not really an identifier for streams in the portal network context, it is identifying all the packets that belong to the same connection. Each uTP socket has one connection ID for sending packets and a different connection ID for receiving packets.

Anyways, I'm happy to rename this if it will improve code readability, what is your opinion @jacobkaufmann?

Reply:
Yea sorry I wasn't making sense by conflating the terms stream and connection.

Passing something non-numeric called `Id` in the same calls as a numeric `Id` is what seemed off to me.

Though looking back, I think I'm splitting hairs. I'm good with leaving it as it is.